CodeContent.jsx

```diff
@@ -1,5 +1,6 @@
 import React,{ useState, useEffect } from "react";
```

Comment:
I would remove the individual imports of useState and useEffect, and just call it by `React.useState` for example, instead of also seperately importing them.